argo-cd-apps/base/all-clusters/infra-deployments/kubelet-config/kubelet.yaml

```diff
@@ -26,13 +26,20 @@ spec:
         targetRevision: main
       destination:
         server: '{{server}}'
+      ignoreDifferences:
+        - group: machineconfiguration.openshift.io
+          kind: KubeletConfig
+          jsonPointers:
+            - /spec/autoSizingReserved
+            - /spec/machineConfigPoolSelector
```

Comment:
humm, I am a bit concerned with that approach. If eventually the thing owning this resource add another field, it will be remove by this config